packages/cozy-viewer/src/Footer/Sharing.jsx

```diff
@@ -18,32 +18,15 @@ const Sharing = ({ file, variant }) => {
       </IconButton>
     ) : (
       <ShareButton
+        className={className}
         fullWidth
         useShortLabel
         docId={file.id}
         onClick={() => setShowShareModal(true)}
       />
     )
 
-  return (
-    <>
-      <SharingProvider
-        client={client}
-        doctype="io.cozy.files"
-        documentType="Files"
-      >
-        {showShareModal && (
-          <ShareModal
-            document={file}
-            documentType="Files"
-            sharingDesc={file.name}
-            onClose={() => setShowShareModal(false)}
-          />
-        )}
-        {SharingButton}
-      </SharingProvider>
-    </>
-  )
+  return <>{SharingButton}</>
```

Comment:
nit: You can return `SharingButton` directly.

Reply:
c'est refactorisé plus tard dans un autre commit 👍